kotlinx.fuzz.test/src/test/kotlin/kotlinx/fuzz/test/SampleTarget.kt

```diff
@@ -0,0 +1,20 @@
+package kotlinx.fuzz.test
+
+import kotlinx.fuzz.KFuzzTest
+import kotlinx.fuzz.KFuzzer
+import kotlinx.fuzz.KFuzzerImpl
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+// TODO: kotlin.test requires a class with empty ctor, NOT an object. Will this be a problem?
```

Comment:
It can be solved in two ways:

1.  By removing `@Test` annotation, we don't need it by design, therefore an actual user will be able to use `object` instead of `class`
2. By adding dependency on `junit-platform-engine` and using ```
tasks.test {
    useJUnitPlatform()
}
```
However I am not sure that it won't break anything, since we use our own engine. But still it helps in replacing `class` with `object`

Reply:
AFAIK (which may be wrong) we only require it to be an object because of [this line](https://github.com/JetBrains-Research/kotlinx.fuzz/blob/5a8fae1bc14ebee50591367b047022b511eb8f04/kotlinx.fuzz.gradle/src/main/kotlin/kotlinx/fuzz/gradle/junit/KotlinxFuzzJunitEngine.kt#L74). @ilma4 what do you think?